ui/app/templates/components/not-found.hbs

```diff
@@ -8,6 +8,9 @@
   </PageHeader>
   <div class="box is-sideless has-background-white-bis has-text-grey has-text-centered">
     <p>Sorry, we were unable to find any content at <code>{{or this.model.path this.path}}</code>.</p>
-    <p>Double check the url or go back <HomeLink @text="home" />.</p>
+    <p>
+      Double check the url or
+      <ExternalLink @href="/" @sameTab={{true}}>go back home</ExternalLink>.
```

Comment:
I used `ExternalLink` here since the `NotFound` component is used in higher level templates where the cluster may not be loaded. This will perform a page refresh at the root url.